package.json

```diff
@@ -4,8 +4,10 @@
   "description": "A reliable way to get env",
   "main": "lib/index.js",
   "scripts": {
-    "test": "apm test",
-    "compile": "ucompiler go"
+    "test": "(apm test) && (flow check) && (eslint . )",
```

Comment:
Was a suggestion, `npm run lint` is somewhat of a standard, but it isn't necessary 😛.